.gitignore

```diff
@@ -2,3 +2,6 @@ target/
 /test-data
 /.bsp
 /image/*.zip
+project/
+.bloop/
+.metals/
```

Comment:
All IDE and dev-environment-specific stuff is better excluded using your `$HOME/.gitignore`

e.g. here's mine:
```bash
$ cat ~/.gitignore
/.priv
.vscode/
.idea/
```

Otherwise project ignore files grow and quickly become an uncontrollable mess.

Reply:
Okay, although this is very debatable, unless we document this in a developers guide or put a comment here. It probably makes sense in an open source project, where there are multiple toolchains possible. In a typical closed project everyone has to use the same environment.